client/src/index.html

```diff
@@ -7,7 +7,11 @@
     <title>Admissions Hub | Galvanize Inc.</title>
     <script type="module" src="./index.jsx"></script>
   </head>
-  <body style="background-color: lightcyan;">
+  <body style="background-color: lightcyan">
     <div id="root"></div>
+    <script src="node_modules/socket.io/client-dist/socket.io.js"></script>
+    <script>
+      var socket = io();
+    </script>
```

Comment:
Just a comment, would we want to see this differed in the head or leave it here at the bottom of the body? I'm not sure differing the script in the head versus scripting at the end of the body is the same thing, maybe if it's a preference thing or if it's an OOO Order of Operations principle. I have no preference I just want to confirm that it is loaded the way we're intending it to.

Reply:
Good catch, this was just an example i ripped from socket.io but I integrated the socket.io-client module into the component so this is unnecessary now

Reply:
removed